www/Dockerfile

```diff
@@ -29,6 +40,24 @@ FROM nginx:mainline-alpine
 # Add dependencies for our scripts
 RUN apk add --no-cache bash openssl inotify-tools
 
+# mountable content for web app (remove default stuff in there)
+RUN rm /usr/share/nginx/html/*
+COPY html/503.html /usr/share/nginx/html
+COPY --from=webapp /usr/src/app/dist /usr/share/nginx/html
```

Comment:
At least the last thing should run last, because it changes most frequently; no need to redo the OpenSSL config when updating webapp contents

Reply:
Redoing openssl config takes <1s, while updateing the webapp takes minutes; hence this order is best for runtime (but not for number of computed layers).

Reply:
@peterthomassen is right - the commands in fact do not compile the webapp. I've undone this change via a fixup commit.